### ISSUE TITLE
Common issues documentation: (1) Networking problems

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#722]
Originally reported of Google Code with ID 666.

```
Hi Michele,

We're creating a Ganeti Common issues doc, and need your help generating content for the topic "Networking problems." Please generate content for section 1 in the Google Doc "Ganeti Eng common issues": https://docs.google.com/a/google.com/document/d/13556aL7_zm-hvF1oLOYiHqlD-MWUgnuyHqGcSHDlfRs/edit. 

This section is a bit more involved than most of the other topics, so definitely feel free to solicit input for any of these sub-topics from other team members.

Deadline: EOD Wednesday, 15 January

Thanks!
```
Originally added on 2014-01-13 13:17:46 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-01-13 17:46:59 +0000 UTC.

Now blocked by: 679 (now #735);

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-10-14 13:27:46 +0000 UTC.

Changed State: WontFix
